findbugs.properties

```diff
@@ -55,7 +55,12 @@ defaults.mavenArtifactId = sonar-findbugs-plugin
 4.2.3.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.3
 4.2.3.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.3/sonar-findbugs-plugin-4.2.3.jar
 4.2.4.description = Use SpotBugs 4.7.3, sb-contrib 7.6.0, and findsecbugs 1.12.0
-4.2.4.sqVersions = [8.9,LATEST]
+4.2.4.sqVersions = [8.9,10.3.*]
 4.2.4.date = 2023-09-29
 4.2.4.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.4
 4.2.4.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.4/sonar-findbugs-plugin-4.2.4.jar
+4.2.5.description = Use SpotBugs 4.8.1, sb-contrib 7.6.0, and findsecbugs 1.12.0
+4.2.5.sqVersions = [8.9,LATEST]
```

Comment:
It's time to start using 9.9 as the range start. :)

Reply:
Hello Ann, that PR is made automatically as part of our release process and I cannot edit.
Can you please close it and have a look at that one instead? https://github.com/SonarSource/sonar-update-center-properties/pull/463
Thanks in advance!